common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -445,7 +451,7 @@ def make_random_password(length=None, choice_fn=random.SystemRandom().choice):
 
 def running(request):
     """Returns True iff request is running a third-party auth pipeline."""
-    return request.session.get('partial_pipeline') is not None  # Avoid False for {}.
+    return request.session.get('partial_pipeline_token') is not None  # Avoid False for {}.
```

Comment:
As above; please switch this to `get(request) is not None`. Pretty much anywhere we used the `request.session.get('partial_pipeline')` meme should be switched over now.